configs/sites/tier1/noaa-aws/README.md

```diff
@@ -1,164 +1,18 @@
 # Provisiong ParallelWorks AWS clusters
 
-## Steps to perform when provisioning a cluster for the very first time
-
-This section provides step by step instructions for installing the basic packages (OS packages, external packages) for spack-stack. Following these steps ensures that the site configuration files in `configs/sites/noaa-aws` work out of the box.
-
-### Basic system packages
-```
-sudo su
-chmod 777 /contrib
-
-# The following three commands were necessary to fix failures accessing these repos
-yum-config-manager --disable intel-clck-2019-repo
-yum-config-manager --disable intel-hpc-platform
-yum-config-manager --disable intelpython
-
-yum install -y qt5-qtbase-devel
-yum install -y qt5-qtsvg-devel
-yum install -y xorg-x11-xauth
-yum install -y xorg-x11-apps
-yum install -y perl-IPC-Cmd
-yum install -y gettext-devel
-yum install -y m4
-exit
-```
-Create a script that can be added to the cluster resource config so that these packages get installed automatically when provisioning new clusters later:
-```
-mkdir -p /contrib/admin
-cat <<EOF > /contrib/admin/basic_setup.sh
-#!/bin/bash
+## Steps to perform to install spack-stack version 1.8.0
 
+sudo su -
 chmod 777 /contrib
-yum install -y qt5-qtbase-devel
-yum install -y qt5-qtsvg-devel
-yum install -y xorg-x11-xauth
-yum install -y xorg-x11-apps
-yum install -y perl-IPC-Cmd
-yum install -y gettext-devel
-yum install -y m4
-EOF
-
-chmod a+x /contrib/admin/basic_setup.sh
-```
-
-### Create a mysql config for local R2D2 use (if applicable)
-```
-sudo su
-cat <<EOF > /contrib/admin/my.cnf
```

Comment:
We may still need to have /contrib/admin/my.cnf saved. It may not need to be done every time a new space for spack stack is configured, i.e., a new /contrib space is prepared